akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -151,11 +118,9 @@ object Sink extends SinkApply {
    * element is dropped and the stream continues.
    *
    * @see [[#mapAsyncUnordered]]
-   */
+   */ // FIXME Why is this method even useful?
```

Comment:
Yeah, so the most useful application is with TCP (or HTTP)

    Tcp().bind(...).foreachParallel(4) { connection => connection.join(handler).run() }

But there might be other uses. Since users might not immediately see that mapAsyncUnordered is perfect for this, (given its signature that says Future, and they might not see how Futures are involved in this case), I think this is a little help that we can give them to use the right tool.